tests/release/release.go

```diff
@@ -0,0 +1,140 @@
+package release
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
```

Comment:
this moved to https://github.com/redhat-appstudio/application-api/tree/main/api/v1alpha1